src/Illuminate/Support/Str.php

```diff
@@ -403,7 +403,7 @@ public static function snake($value, $delimiter = '_')
     public static function startsWith($haystack, $needles)
     {
         foreach ((array) $needles as $needle) {
-            if ($needle != '' && mb_strpos($haystack, $needle) === 0) {
+            if ($needle != '' && substr($haystack, 0, strlen($needle)) === $needle) {
```

Comment:
Got it, great change then :)

One issue though, `$needle` should be cast to string. I'll make a PR for this tonight.